src/tests/ftest/pool/BadCreate.py

```diff
@@ -27,12 +27,9 @@
 import traceback
 import sys
 import json
-from avocado import Test, main
+from apricot import Test
+from avocado import main
```

Comment:
(pylint-import-error) Unable to import 'avocado'

(pylint-unused-import) Unused main imported from avocado